### PR TITLE
[Fixes #100417804] DM CSS style for the Remote Service Applications

### PR DESCRIPTION
**Pull request by Sean-Stellingwerff**

### CODE REVIEW

**Comment by Sean-Stellingwerff** [comment]
Please note that all sub-applications (e.g. firmware, crashlogs, offline, system) should be thoroughly tested. I have not tested if adding / deleting firmwares works, for example, because I am unsure if that would influence live systems.

File: app/views/firmware/new.erb [hunk 1]
```diff
@@ -1,47 +1,51 @@
...
-
-<% end %>
\ No newline at end of file
+    <div class="field">
+      <%= f.label :description, 'Description' %>
+      <%= f.text_area :description, :rows => 4, :cols => 100 %>
+      <p class="help">Write a short (public) release note</p>
+    </div>
+  
+    <p>WARNING: if you add this firmware version, Store!D Units will IMMEDIATELY be able to install this firmware version.</p>
```
Comment (emilkappert) [inline app/views/firmware/new.erb]: Store!D? Should this be Renos?

File: app/views/offline/index.html.erb [hunk 2]
```diff
@@ -1,72 +1,71 @@
...
       <tr>
         <td>Username</td>
         <td>
           <span class="otp-username">
             <% if @otp_response.present? %>
-              <%=  @otp_response['username'] %>
+              <%= @otp_response['username'] %>
             <% else %>
               <%= "-" %>
             <% end %>
```
Comment (emilkappert) [inline app/views/offline/index.html.erb]: This could easily be a one-liner

File: app/views/system/_show_connected.erb [hunk 3]
```diff
@@ -1,27 +1,37 @@
...
-<p>
-  Connected since: <strong><%= @device.remote_management_connected_since.to_s %></strong><br />
-  Automatic disconnect in: <strong><span id="timeout_seconds"><%= @device.remote_management_timeout_seconds %></span> seconds</strong>
-</p>
-
-<p>
 <% if @current_user_connected %>
   <% if @device.remote_management_type == "ost" %>
-    You can now connect OST Manager to hostname <strong>remoteost.nedap-cube.net</strong> port <strong><%= @device.remote_management_port %></strong>
+    <p>You can now connect OST Manager to hostname <strong>remoteost.nedap-cube.net</strong> port <strong><%= @device.remote_management_port %></strong></p>
```
Comment (emilkappert) [inline app/views/system/_show_connected.erb]: Ask Bas for the correct URL

**Pull request merged**